plugin/trino-mongodb/src/test/java/io/trino/plugin/mongodb/TestMongoConnectorTest.java

```diff
@@ -721,6 +722,19 @@ public void testNativeQuerySimple()
                 .matches("SELECT * FROM region");
     }
 
+    @Test
+    public void testNativeQueryAmbiguousTableName()
+    {
+        List<String> caseInsensitiveSameCollections = ImmutableList.of("AmbiCol", "AMBICOL", "ambicol");
+
+        for (String collection : caseInsensitiveSameCollections) {
+            client.getDatabase("tpch").getCollection(collection).insertOne(new Document("a", "b"));
+        }
+
+        assertQueryFails("SELECT * FROM ambicol",
```

Comment:
This isn't native query.

Reply:
I've renamed it to `testSelectAmbiguousTableName`